pkg/sloop/webserver/webfiles/debug.js

```diff
@@ -0,0 +1,13 @@
+/*
+ * Copyright (c) 2019, salesforce.com, inc.
```

Comment:
```suggestion
 * Copyright (c) 2021, salesforce.com, inc.
```

Do we have Copyright hook that checks Copyrights for updated and new files in the pull-requests ?